account_invoice_open_sale/models/account_invoice.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+# © 2016 Opener B.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools.translate import translate, _
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    has_sale_order = fields.Boolean(compute='_compute_has_sale_order')
+
+    @api.multi
+    def _compute_has_sale_order(self):
```

Comment:
Please add a simple test for this field value (hopefully you can find invoices in the demo data with and without a related sale order.